src/middle-end/auth/index.js

```diff
@@ -1,6 +1,135 @@
-module.exports = {
-    actions: require('./actions'),
-    reducer: require('./reducer'),
-    selectors: require('./selectors'),
-    initialize: require('./init')
+const MiddleEnd = require('strange-middle-end');
+const { ROLES } = require('../../utils/constants');
```

Comment:
We're actually not using any roles in this example; should we remove them from `utils/constants`, or leave them as a starting place?

Reply:
I think we should nix them if they don't have any functional use here.